packages/react/src/components/header/components/navigationLanguageSelector/NavigationLanguageSelector.module.scss

```diff
@@ -31,7 +34,11 @@
   padding: 0;
   white-space: nowrap;
 
-  @include header.element-focus-state;
+  &:focus-visible {
+    border: none;
+    outline: var(--header-focus-outline-width) solid var(--color-coat-of-arms);
+    outline-offset: var(--header-focus-outline-width);
+  }
```

Comment:
I couldn't find the designs for language bar item focus but I think the current focus style is bit too tall, especially compared to other items. The focus style in header items is quite compact overall.
<img width="201" alt="Screenshot 2023-08-16 at 8 30 40" src="https://github.com/City-of-Helsinki/helsinki-design-system/assets/108321134/ddc568ce-b799-4564-8b24-72db10371248">

Reply:
And on mobile view the focus goes over the language bar. 
<img width="252" alt="Screenshot 2023-08-16 at 8 25 44" src="https://github.com/City-of-Helsinki/helsinki-design-system/assets/108321134/f061bf79-41de-418c-912f-9e9e6876af08">